### PR TITLE
Removed vertical slider code

### PR DESCRIPTION
**Pull request by williamjameshandley**
# Description

The [vertical option for slider widgets](https://github.com/matplotlib/matplotlib/pull/10746) 
is now in the stable version of matplotlib, so we no longer need to replicate
it. 

This PR was prompted (via a cron job test) by an update to pydocstyle that
flagged that the now unnecessary code did not have compliant docstrings. I
propose to resolve the issue by removing the code

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My code is PEP8 compliant (`flake8 anesthetic tests`)
- [x] My code contains compliant docstrings (`pydocstyle --convention=numpy anesthetic`)
- [x] New and existing unit tests pass locally with my changes (`python -m pytest`)
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by williamjameshandley** [comment]
The tests pass on python 3.6, 3.7 and 3.8. I propose to take this opportunity to drop support for 3.5 and 2.7, to both speed up the test cycle, and to recognise the [imminent deprecation](https://pythonclock.org/).

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,6 @@ python:
     - "3.8"
     - "3.7"
     - "3.6"
-    - "3.5"
-    - "2.7"
```
Comment (lukashergt) [inline .travis.yml]: These python versions should also be taken out of the `setup.py` file and the `README.rst` should be updated accordingly.

Comment (williamjameshandley) [reply .travis.yml]: Well spotted.

Comment (lukashergt) [reply .travis.yml]: stumbled over a python35 environment which didn't work with anesthetic anymore...

Comment (williamjameshandley) [reply .travis.yml]: Can you remember what it was that didn't work?

Comment (lukashergt) [reply .travis.yml]: The newest matplotlib (v3.1.2) which incorporates the vertical slider has dependency python>=3.6. 

Maybe worth incorporating in the requirements? i.e.
```
matplotlib>=3.1.2
```

Comment (williamjameshandley) [reply .travis.yml]: Added as part of #73

**Pull request merged**